sdk-python/omnilogger/pricing/storage_manager.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+from typing import Any
+
+import platformdirs
+
+
+def pricing_reference_path() -> str:
+    return os.path.join(os.path.dirname(__file__), "pricing.json")
+
+
+def pricing_config_path() -> str:
```

Comment:
So how is this working? If it's a test, we just using the file path and find the test `pricing.json`? If it's not, we create the directory if it doesn't exist?

Reply:
happy to go through this with you

Reply:
This function returns the local storage path of the `pricing.json` and creates it if the folder for our package doesn't exist yet.

Reply:
=> discussion recap: the issue is not how this function works, but how the pricing class has a different behaviour for testing. The way to go is to remove the is_test argument and mock the pricing_config_path during testing